spec/unit/module_tool/applications/installer_spec.rb

```diff
@@ -32,6 +34,12 @@
     installer.stubs(:module_repository).returns(remote_source)
   end
 
+  if Puppet.features.microsoft_windows?
+    before :each do
+      Puppet.settings[:module_working_dir] = Dir.mktmpdir('installertmp')
```

Comment:
We have talked recently about making settings immutable after first load, so this change would add tech debt that later needs to be cleaned up... is there another better way that we could do this? (Stubbing the value with a test framework would be better if that's possible - but if there are other better solutions that would be preferred)